Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -348,17 +351,40 @@ public class EOSSessionsManager : IEOSSubManager
         private SessionDetails JoiningSessionDetails = null;
         private ulong JoinedSessionIndex = 0;
 
+        /// <summary>
+        /// When joining a Session you did not create, the Session's name is not shared with you.
+        /// Instead the Session is given a local name, starting with this, when <see cref="GenerateJoinedSessionName(bool)"/> is called.
+        /// This local name doesn't have to follow any particular formatting, but does need to be locally unique and consistent.
+        /// </summary>
         private const string JOINED_SESSION_NAME = "Session#";
         private const uint JOINED_SESSION_NAME_ROTATION_NUM = 9;
         private const int JOINED_SESSION_NAME_ROTATION = 9;
+
+        /// <summary>
+        /// The Bucket Id to create Sessions under, and to search by when finding Sessions.
+        /// Bucket Ids are used to separate search results, operating as a major filter option.
+        /// Your application could use different Bucket Ids for Sessions. If you do,
+        /// then you will need to search by those Bucket Ids in order to find those Sessions online.
+        /// 
+        /// https://dev.epicgames.com/docs/en-US/api-ref/functions/eos-session-modification-set-bucket-id
+        /// </summary>
         private const string BUCKET_ID = "SessionSample:Region";
 
+        /// <summary>
+        /// When creating a Session or searching for one, this is the attribute used to filter for buckets.
+        /// In these sample functions, <see cref="CreateSession(Session, bool, Action)"/> sets the BucketId
+        /// in <see cref="CreateSessionModificationOptions"/> as well as an <see cref="SessionAttribute"/>.
+        /// </summary>
         private const string EOS_SESSIONS_SEARCH_BUCKET_ID = "bucket";
+
+        // TODO: All three of these are unused, remove?
         private const string EOS_SESSIONS_SEARCH_EMPTY_SERVERS_ONLY = "emptyonly";
         private const string EOS_SESSIONS_SEARCH_NONEMPTY_SERVERS_ONLY = "nonemptyonly";
         private const string EOS_SESSIONS_SEARCH_MINSLOTSAVAILABLE = "minslotsavailable";
 
         // UI Parameterized Callbacks
+        // TODO: None of these are subscribed to, several of them aren't implemented fully. Remove? Implement more fully?
+        // UIOnSessionCreated is actually utilized
```

Comment:
I'd be more specific with respect to which you mean when you say, "these," particularly since you follow up to specify one in particular. Simpler would be to just limit the note to specifically reference the ones it applies to, or be more verbose.

Reply:
Specified. I decided the clearest way to do that was to put `UIOnSessionCreated` in to its own singular spot, make a line break, and then comment about the following being unimplemented.